data/processing.py

```diff
@@ -712,15 +806,21 @@ def latest_reports():
     }
   }
 
-  return [https_report, analytics_report]
+  a11y_domains = Domain.eligible('a11y')
+  a11y_report = {'a11y': {}}
+  for domain in a11y_domains:
+    a11y_report['a11y'][domain['domain']] = domain['a11y']['error_details']
+
+  return [https_report, analytics_report, a11y_report]
 
 # Hacky helper - print out the %'s after the command finishes.
 def print_report():
   print()
 
   report = Report.latest()
   for report_type in report.keys():
-    if report_type == "report_date":
+    # The a11y report has a very different use than the others
+    if report_type == "report_date" or report_type == "a11y":
```

Comment:
The a11y report is basically just a full dump of the scan data, with line by line errors per domain. The others are high level summaries.